### ISSUE TITLE
torch.optim.lr_scheduler

### ISSUE THREAD

**Issue by zhjw0927** [pytorch/pytorch#31286]
I have two questions about optim:
1. scheduler.step() need parameters?  Before, I can run the code with not have parameters, but now remind me of need a parameter. I'm confused about this.
2. At the start of each epoch, first calling optimizer.step() in train, then calling scheduler.step(). This is update of 1.1. This means lr update after training. Why do you do this?

**Comment by zhjw0927** [comment]
I'sorry, please ignore first qusetion. I got it wrong.

**Comment by zou3519** [comment]
Please direction questions to our forums: discuss.pytorch.org. We like to keep our issue tracker filled with bug reports and feature requests.
